specification/introduction/index.md

```diff
@@ -23,7 +23,7 @@ submit data and metadata to digital repositories; store it in long-term preserva
 
 This Common Specification for Information Packages aims to summarise the common aspects of all these IPs. The main goal in the development of this specification has been to identify and standardise the common aspects of IPs which are equally relevant and implemented by any of the functional entities of the overall digital preservation process presented in OAIS (i.e. pre-ingest, ingest, archival storage, data management and access). The practical implementation is that the specification therefore allows for the development of generic tools and code libraries which can either be applied commonly across the whole lifecycle of digital data, or be reused as the basis for developing more specific, content or process-aware tools.
```

Comment:
Do we aim to summarise the common aspects? Where's the long versions if these are the summaries?

Reply:
They are in the OAIS reference model, but the sentance is a repeat of the one after it.